rpcclient/errors.go

```diff
@@ -338,6 +338,17 @@ func (r BitcoindRPCErr) Error() string {
 	return "unknown error"
 }
 
+// BitcoindErrMap is a map of additional errors bitcoind can throw that are
+// version dependent (e.g. versions up to v29 return the error as specified in
+// `Error()` above, while versions v30 and beyond return the error as mapped
+// here.
+var BitcoindErrMap = map[string]error{
+	// The error message was changed in
+	// https://github.com/bitcoin/bitcoin/pull/33050 which will be included
+	// in bitcoind v30.0 and beyond.
+	"mempool script verify flag failed": ErrNonMandatoryScriptVerifyFlag,
```

Comment:
can you also add: `block-script-verify-flag-failed` see https://github.com/bitcoin/bitcoin/blob/master/doc/release-notes-33183.md ?

Reply:
Ah, yes. Added.